FromUnixPipe.spec.ts

```diff
@@ -0,0 +1,28 @@
+import 'es6-shim';
+import 'reflect-metadata';
+import * as moment from 'moment';
+import {FromUnixPipe} from './FromUnixPipe';
+import {DateFormatPipe} from './DateFormatPipe';
+
+describe('FromUnixPipe', () => {
+  describe('#transform', () => {
+    it('should parse a unix timestamp number to moment', () => {
+      const pipe = new FromUnixPipe();
+      const result = pipe.transform(1456263980);
+      expect(result).toEqual(moment.unix(1456263980));
+    });
+
+    it('should parse a unix timestamp string to moment', () => {
+      const pipe = new FromUnixPipe();
+      const result = pipe.transform('1456263980');
+      expect(result).toEqual(moment.unix(1456263980));
+    });
+
+    it('should format a unix timestamp', () => {
+	  const unixPipe = new FromUnixPipe(),
+            datePipe = new DateFormatPipe();
+	  const result = datePipe.transform(unixPipe.transform(1456263980), ['hh:mmA']);
+      expect(result).toBe('01:46PM');
```

Comment:
Thanks! This test is not required, as it is beyond the scope of just testing unixPipe. Also, it will fail in different timezones